community/slack_channel_guide.md

```diff
@@ -38,20 +39,23 @@ Want to add/change something? Give a PR.
 | #git | git questions can go here |
 | #help | Ask us anything (probably about tech, but maybe not? I don't know!) Yes, even if you're new to all this, so are lots of people. |
 | #ios | iOS related development |
+| ##infrastructure-team | Discuss infrastructure-team related topic here!|
```

Comment:
extra `#` here

Reply:
@Kohdz when I said "extra `#` here" I meant that you need to remove the extra `#`, so it should be `#infrastructure-team` rather than `##infrastructure-team`